worker.yaml

```diff
@@ -64,6 +64,8 @@ plugins:
     volumes:
 #      cifs: {}
 
-  network:
-    tinc:
-      enabled: true
+  tinc:
+    enabled: true
+  l2tp:
+    enabled: true
```

Comment:
Looks strange. I'd rather make `l2tp` optional field. For example `l2tp: {}` configures the driver, otherwise skips.

Reply:
i have to talk to my manager